packages/shared/components/inputs/OptionalInput.svelte

```diff
@@ -62,14 +63,19 @@
         {...$$restProps}
     />
     {#if !isOpen}
-        <AddInputButton
-            bind:buttonElement
-            bind:open={isOpen}
-            text={label}
-            onClick={open}
-            {onMouseEnter}
-            {onMouseLeave}
-        />
+        <button
+            bind:this={buttonElement}
+            class="py-1.5 px-3 w-max bg-gray-100 hover:bg-gray-200 focus:bg-gray-200 dark:bg-gray-700 dark:hover:bg-gray-900 dark:focus:bg-gray-900 text-gray-600 dark:text-gray-500 rounded-md"
```

Comment:
Once the popup is opened and closed, the hover stays on the button. I think this already happened in develop, but maybe it can be improved here. wdyt?
![image](https://github.com/iotaledger/firefly/assets/100352899/ce8fc8fa-46cc-4545-8de9-63ed64c34936)